test/e2e/webhook_test.go

```diff
@@ -424,3 +426,1021 @@ var _ = Describe("webhook tests for MC taints", Ordered, func() {
+										Path:     "/meta/labels/test-key",
+									},
+								},
+							},
+						},
+					},
+				},
+			}
+			By(fmt.Sprintf("expecting denial of CREATE override %s", cro.Name))
+			err := hubClient.Create(ctx, cro)
+			var statusErr *k8sErrors.StatusError
+			Expect(errors.As(err, &statusErr)).To(BeTrue(), fmt.Sprintf("Create CRO call produced error %s. Error type wanted is %s.", reflect.TypeOf(err), reflect.TypeOf(&k8sErrors.StatusError{})))
+			Expect(statusErr.Status().Message).Should(MatchRegexp("only labelSelector is supported"))
+			return nil
+		}, testutils.PollTimeout, testutils.PollInterval).Should(Succeed())
+	})
+
+	It("should deny create CRO with invalid JSON patch override", func() {
+		Eventually(func(g Gomega) error {
+			cro := &placementv1alpha1.ClusterResourceOverride{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: croName,
+				},
+				Spec: placementv1alpha1.ClusterResourceOverrideSpec{
+					ClusterResourceSelectors: []placementv1beta1.ClusterResourceSelector{
+						{
+							Group:   "rbac.authorization.k8s.io/v1",
+							Kind:    "ClusterRole",
+							Version: "v1",
+							Name:    "test-cluster-role",
+						},
+					},
+					Policy: &placementv1alpha1.OverridePolicy{
+						OverrideRules: []placementv1alpha1.OverrideRule{
+							{
+								ClusterSelector: &placementv1beta1.ClusterSelector{
+									ClusterSelectorTerms: []placementv1beta1.ClusterSelectorTerm{},
+								},
+								JSONPatchOverrides: []placementv1alpha1.JSONPatchOverride{
+									{
+										Operator: placementv1alpha1.JSONPatchOverrideOpRemove,
+										Path:     "/meta/labels/test-key",
+										Value:    apiextensionsv1.JSON{Raw: []byte(`"test-value"`)},
+									},
+									{
+										Operator: placementv1alpha1.JSONPatchOverrideOpReplace,
+										Path:     "/kind",
+										Value:    apiextensionsv1.JSON{Raw: []byte(`"new-kind"`)},
+									},
+									{
+										Operator: placementv1alpha1.JSONPatchOverrideOpReplace,
+										Path:     "////",
+										Value:    apiextensionsv1.JSON{Raw: []byte(`"new-kind"`)},
+									},
+								},
+							},
+						},
+					},
+				},
+			}
+			By(fmt.Sprintf("expecting denial of CREATE override %s", cro.Name))
+			err := hubClient.Create(ctx, cro)
+			var statusErr *k8sErrors.StatusError
+			Expect(errors.As(err, &statusErr)).To(BeTrue(), fmt.Sprintf("Create CRO call produced error %s. Error type wanted is %s.", reflect.TypeOf(err), reflect.TypeOf(&k8sErrors.StatusError{})))
+			Expect(statusErr.Status().Message).Should(MatchRegexp("remove operation cannot have value"))
+			Expect(statusErr.Status().Message).Should(MatchRegexp("cannot override typeMeta fields"))
+			Expect(statusErr.Status().Message).Should(MatchRegexp("path cannot contain empty string"))
+			return nil
+		}, testutils.PollTimeout, testutils.PollInterval).Should(Succeed())
+	})
+
+	It("should deny create CRO for a resource already selected by another CRO", func() {
```

Comment:
can you also move this test out? this test creates a cro and may impact other tests. it cannot run in parallel with others.